ha/alert/node_alert_monitor.py

```diff
@@ -32,4 +32,33 @@ def process_alert(self):
         Log.debug("Processing event for NodeAlertMonitor")
         # Environment variable are avilable in self.crm_env
         self.iem = IemGenerator()
-        self.iem.generate_iem(self.crm_env["CRM_alert_node"], self.alert_event_module, self.alert_event_type)
+        self.alert_dup = HandleAlertDuplication()
+        self.nodes_ids, self.local_node_id, self.local_node_name = self.alert_dup.get_node_id()
+        # Generate and send IEM only through the highest online node in cluster.
+        if self.nodes_ids[-1] == self.local_node_id:
+            self.iem.generate_iem(self.crm_env["CRM_alert_node"], self.alert_event_module, self.alert_event_type)
+            Log.info(f"Sent IEM alert from local node: {self.local_node_name} / {self.local_node_id}")
+        else:
+            Log.info("Not the highest node to sent IEM node failure alert.")
+
+class HandleAlertDuplication:
```

Comment:
This class is not required. Add _get_all_nodes and _get_local_node in previous class.